packages/browser/src/authenticatedFetch/bearer/BearerAuthenticatedFetcher.ts

```diff
@@ -44,7 +44,6 @@ export default class BearerAuthenticatedFetcher
   implements IAuthenticatedFetcher {
   constructor(
     @inject("fetcher") private fetcher: IFetcher,
-    @inject("urlRepresentationConverter")
```

Comment:
For my education - what was the bug here that required you to remove this? Is this part of the whole "we're using three different types of URLs everywhere" hubbub?

Reply:
No, this was literally just being reported as a runtime error when this code was being executed. The problem never surfaced before, as this code was never actually being executed. Instead the DpopAuthenticatedFetcher code was being used - but I switched from that to the 'Bearer' version instead (since doing DPoP is for 'later'). Simple as that really :) !